src/tools/create-logger.js

```diff
@@ -11,6 +11,18 @@ const constants = require('../constants');
 
 const truncate = str => dataTools.simpleTruncate(str, 3500, ' [...]');
 
+const formatHeaders = (headers = {}) => {
+  if (_.isString(headers)) {
+    // we had a bug where headers coming in as strings weren't getting censored. If something calls this with stringified headers, we'll bow out. Pass the raw object instead.
+    return 'ERR - refusing to log possibly uncesored headers';
+  }
+  return Object.entries(headers)
```

Comment:
Is there an advantage to using `Object.entries(headers).map` over `_.map(headers)`? FWIW I think the equivalent to this would be

```
_.map(headers, (value, header) => `${header}: ${value}`)
```

Reply:
I like lodash a lot but It's also nice to use standard language features when available. ¯\\\_(ツ)_/¯